docs/_includes/confluence.adoc

```diff
@@ -0,0 +1,70 @@
+////
+Header: Convert Confluence XHTML to Asciidoctor
+
+Included in:
+
+- user-manual
+////
+
+You can convert Atlassian Confluence XHTML pages to Asciidoctor using this http://www.groovy-lang.org/download.html[Groovy] script.
+
+The script calls the http://pandoc.org/[Pandoc] tool to convert single or multiple HTML files exported from Confluence to AsciiDoc files (using the Python version of the AsciiDoc toolchain).
```

Comment:
> Restructuring the content into a tooling guide is a good move but requires another issue IMHO.

True. Let's proceed with where it is now and open an issue to move migration content into a dedicated guide.

>  I think that it will be complete once I remove the statement about python AsciiDoc.

Correct.